promlog/log.go

```diff
@@ -14,6 +14,8 @@
 // Package promlog defines standardised ways to initialize Go kit loggers
 // across Prometheus components.
 // It should typically only ever be imported by main packages.
+
+// Deprecated: This package has been replaced with github.com/prometheus/common/promslog.
```

Comment:
I'm wondering if we should wait one release of `prometheus/common` before adding the deprecation.

We have a lot of work to do to migrate, and we have linters in place that will complain about deprecated uses.